conf/config-schema.json

```diff
@@ -0,0 +1,16 @@
+{
+    "type": "object",
+    "properties": {
+        "root": { "type": "boolean" },
+        "globals": { "type": ["object", "array"] },
+        "global": { "type": ["object", "array"] },
```

Comment:
Does `globals` allow an array?
I didn't find it in the doc: http://eslint.org/docs/user-guide/configuring#specifying-globals